### PR TITLE
Feat mobile authj wireup

### PR DESCRIPTION
**Pull request by kitorer**
~~I just need to test this and I will PR this up~~
I made some changes to the styling of the register and login, just centering views and expanding the pixel length of the text input
todo: Raciel has toast error messages on web, we will need to get those too

### CODE REVIEW

File: mobile/App.tsx [hunk 1]
```diff
@@ -15,19 +17,24 @@ export default function App() {
...
+          options={{ headerShown: false }}
         />
-        <Stack.Screen 
+        <Stack.Screen
           name="Register"
           component={Register}
-          options={{headerShown: false}}
+          options={{ headerShown: false }}
+        />
+        <Stack.Screen
```
Comment (kitorer) [inline mobile/App.tsx:32]: this page will eventually be refactored with a auth flow in mind [[](https://reactnavigation.org/docs/auth-flow)](https://reactnavigation.org/docs/auth-flow)

File: mobile/src/pages/Login.tsx [hunk 2]
```diff
@@ -1,58 +1,87 @@
...
+          />
         </View>
-    );
+        <Pressable className="items-center justify-center">
+          <Text className="text-xl my-10 text-indigo-900 font-bold">
+            Forgot your password?
+          </Text>
+        </Pressable>
+      </View>
+      <View className="flex-1 w-full items-center position-relative justify-center">
```
Comment (kitorer) [inline mobile/src/pages/Login.tsx:57]: api wireup changes start below, sorry rest is prettier

File: mobile/src/pages/Register.tsx [hunk 3]
```diff
@@ -1,79 +1,96 @@
...
+              className={inputClassName}
+              secureTextEntry={true}
+              placeholder="*Password"
+              placeholderTextColor="#000000"
+              onChangeText={setConfirmPassword}
+            />
+          </View>
+        </View>
+        <View className="flex-1 w-full items-center my-12 justify-center">
+          <Pressable>
```
Comment (kitorer) [inline mobile/src/pages/Register.tsx:75]: api wireup changes start below

**Review by rob-3 — CHANGES_REQUESTED** [review]

File: mobile/src/pages/Login.tsx [hunk 4]
```diff
@@ -57,16 +58,20 @@ export default function Login({ navigation }) {
...
             className="mb-10"
+            disabled={isPending}
             onPress={() => {
+              setIsPending(true);
               login(username, password)
                 .then(async (data) => {
                   if (data) {
                     storeAuthSession(data);
                     await navigation.navigate("FeedPage");
+                    setIsPending(false);
```
Comment (rob-3) [inline mobile/src/pages/Login.tsx:69]: I think this should go above `navigation.navigate()` because the component might be destroyed when that happens.

**Review by rob-3 — APPROVED** [review]

**Pull request merged**